### DISCUSSION TITLE
[Bug] Backend: custom required variant field

### DISCUSSION THREAD

**Discussion by josias-hosang** [swellstores/community#345] [Help] [answered]
I added a custom int-field in product.variants. Set it to required with fallback value. When I try to create a product with options generating variants, whole variants section is missing and can't be generated. Shouldn't this be possible?

**Comment by logeshswell** [comment, marked as answer]
@josias-hosang I think it is expected because variant creation will throw a validation error due to a missing value in custom int field.
Can you disable the required toggle for that field and try it?
